README.md

```diff
@@ -179,7 +179,9 @@ _See documentation [here](#xgboost)._
 $ aim up
 ```
 
-## Roadmap 
+## Roadmap
+
+:sparkle: View the [Aim product roadmap](https://github.com/orgs/aimhubio/projects/3)
 
```

Comment:
I'd make them two subsections. Something like this:

# Roadmap
## Detailed Sprints
:sparkle: The [Aim product roadmap](https://github.com/orgs/aimhubio/projects/3)

- The `Backlog` contains the issues we are going to choose from and prioritize weekly 
- We prioritize the issues into weekly sprints based on the user-feedback and the most requested issues

## High-level roadmap
The high-level features we are going to work on the next few months
...